scss/mixin.scss

```diff
@@ -43,3 +43,20 @@
     }
   }
 }
+
+@mixin form-control( $border-color ) {
```

Comment:
I'm confused by this- `.form-control` is a class bootstrap gives us, so making a mixin with the same name that may or may not override things is...hard to understand. If we don't want this styling applied to everything, would this better be suited to an additional class on certain elements?

Reply:
the main difference here is the border color. would it be easiest if these styles were just moved to `.form-control` and border-color was overrided specifically on the search and dropdown elements?

Reply:
Overriding bootstrap's `.form-control` at a project level is a better solution here if we want all these styles to apply to all form elements. If we just want them to apply to these elements, at least pick a different name?

Reply:
👍